pkg/storage/replica.go

```diff
@@ -3502,7 +3502,7 @@ func (r *Replica) processRaftCommand(
 	var requestedLease roachpb.Lease
 	var ts hlc.Timestamp
 	var rSpan roachpb.RSpan
-	if raftCmd.ReplicatedEvalResult != nil {
+	if idKey != "" {
```

Comment:
wasn't the old code

```go
if true {
  // still here
} else if idKey != "" {
  // you deleted this
}
```

and why isn't the new code just the `// still here` part?